### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by picturepan2**
- [ ] Update both Chinese translations, and fix missing ones.

### CODE REVIEW

File: src/translations/zh-hans.global.json [hunk 1]
```diff
@@ -169,7 +172,8 @@
                 },
                 "alertBanner": {
                     "title": "请勿关闭此窗口！"
-                }
+                },
+                "fundWith": "打开"
```
Comment (think-in-universe) [inline src/translations/zh-hans.global.json:176]: `Fund with` should be `支付` ? I don't know exactly where the button is, so may be wrong.

File: src/translations/zh-hans.global.json [hunk 2]
```diff
@@ -478,7 +482,7 @@
         "recoverAccount": "恢复账户",
         "goBack": "返回",
         "createNewAccount": "创建新的账户",
-        "addAccount": "添加其他账户",
+        "importAccount": "添加其他账户",
```
Comment (think-in-universe) [inline src/translations/zh-hans.global.json:485]: `导入其他账户` is preferred if follow the change precisely, but `添加其他账户` is also fine.

File: src/translations/zh-hant.global.json [hunk 3]
```diff
@@ -169,7 +172,8 @@
                 },
                 "alertBanner": {
                     "title": "請勿關閉此窗口！"
-                }
+                },
+                "fundWith": "打開"
```
Comment (think-in-universe) [inline src/translations/zh-hant.global.json:176]: same as above for Simplified Chinese

File: src/translations/zh-hant.global.json [hunk 4]
```diff
@@ -478,7 +482,7 @@
         "recoverAccount": "恢復賬戶",
         "goBack": "返回",
         "createNewAccount": "創建新的賬戶",
-        "addAccount": "添加其他賬戶",
+        "importAccount": "添加其他賬戶",
```
Comment (think-in-universe) [inline src/translations/zh-hant.global.json:485]: same as above for Simplified Chinese

**Review by think-in-universe — COMMENTED** [review]
added some comments about two strings

File: src/translations/zh-hans.global.json (hunk 1 above)
Comment (picturepan2) [reply src/translations/zh-hans.global.json:176]: 通过 MoonPay 支付的按钮，但是 MoonPay 是图标，「打开 MoonPay」比较合适。

File: src/translations/zh-hans.global.json (hunk 2 above)
Comment (picturepan2) [reply src/translations/zh-hans.global.json:485]: 「导入已有账户」吧

**Pull request merged**